rambo/app.py

```diff
@@ -344,6 +334,9 @@ def scp(ctx=None, locations=None):
 
     locations = [copy_from, copy_to]
 
+    if utils.read_specs()['provider'] == 'gce':
```

Comment:
I'm guessing the is s temp solution type thing, but It looks like something that might need to be thought out more. (specifically the 'read_specs' bit)